.circleci/config.yml

```diff
@@ -70,9 +82,46 @@ jobs:
       - run: make build
       - check-uncommitted
       - *save_cache
+      - persist_to_workspace:
+          root: *workspace_root
+          paths:
+            - .
+
+  deploy:
+    executor:
+      name: deploy
+    working_directory: *workspace_root
+    steps:
+      - run:
+          name: Install Serverless CLI and dependencies
+          command: sudo npm i -g serverless
+      - *attach_workspace
+      - run:
+          name: Deploy Function
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+                export AWS_ACCESS_KEY_ID=$AWS_STAGING_ACCESS_KEY_ID
+                export AWS_SECRET_ACCESS_KEY=$AWS_STAGING_SECRET_ACCESS_KEY
+                sudo -E sls deploy function -f server --stage staging
+            elif [ "${CIRCLE_BRANCH}" == "production" ]; then
+                export AWS_ACCESS_KEY_ID=$AWS_PRODUCTION_ACCESS_KEY_ID
+                export AWS_SECRET_ACCESS_KEY=$AWS_PRODUCTION_SECRET_ACCESS_KEY
+                sudo -E sls deploy function -f server --stage production
```

Comment:
Maybe reuse `make deploy-lambda-fast` here?

Reply:
Right now, that target also clean builds, but I'm running this in the node container without the infrastructure needed to do same, and relying on workspace artifacts instead. Otherwise, I'd definitely agree!

Reply:
Good catch. Missed that :+1: